src/Common/ProfileEvents.cpp

```diff
@@ -190,8 +191,10 @@
     \
     M(ReplicaPartialShutdown, "How many times Replicated table has to deinitialize its state due to session expiration in ZooKeeper. The state is reinitialized every time when ZooKeeper is available again.") \
     \
+    M(SelectedPartsByPartitionKey, "Number of data parts selected by partition in a MergeTree table.") \
```

Comment:
Users will wonder why the generally more useful and general metrics (SelectedParts) is listed after a more specialized metric (SelectedPartsByPrimaryKey). 

Same in l. 197/198.

I know that the description (l. 194) was updated, but I still don't understand (and users will not understand too), what the metric represents. Especially "ByPartitionKey" is confusing, does that mean "ByPrimaryKey"??

Reply:
I didn't come up with a better way to describe SelectedPartsByPartitionKey, deleted it for now.